openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1247,9 +1247,9 @@ This specification defines the following Credential Issuer Metadata:
 * `credential_endpoint`: REQUIRED. URL of the Credential Issuer's Credential Endpoint. This URL MUST use the `https` scheme and MAY contain port, path, and query parameter components.
 * `batch_credential_endpoint`: OPTIONAL. URL of the Credential Issuer's Batch Credential Endpoint. This URL MUST use the `https` scheme and MAY contain port, path, and query parameter components. If omitted, the Credential Issuer does not support the Batch Credential Endpoint.
 * `deferred_credential_endpoint`: OPTIONAL. URL of the Credential Issuer's Deferred Credential Endpoint. This URL MUST use the `https` scheme and MAY contain port, path, and query parameter components. If omitted, the Credential Issuer does not support the Deferred Credential Endpoint.
-* `credential_response_encryption_alg_values_supported`: OPTIONAL. Array containing a list of the JWE [@!RFC7516] encryption algorithms (`alg` values) [@!RFC7518] supported by the Credential and/or Batch Credential Endpoint to encode the Credential or Batch Credential Response in a JWT [@!RFC7519].
-* `credential_response_encryption_enc_values_supported`: OPTIONAL. Array containing a list of the JWE [@!RFC7516] encryption algorithms (`enc` values) [@!RFC7518] supported by the Credential and/or Batch Credential Endpoint to encode the Credential or Batch Credential Response in a JWT [@!RFC7519].
-* `require_credential_response_encryption`: OPTIONAL. Boolean value specifying whether the Credential Issuer requires additional encryption on top of TLS for the Credential Response and expects encryption parameters to be present in the Credential Request and/or Batch Credential Request, with `true` indicating support. When the value is `true`, `credential_response_encryption_alg_values_supported` parameter MUST also be provided.  If omitted, the default value is `false`.
+* `credential_response_encryption`: OPTIONAL. Object containing information whether the Credential Issuer requires encryption of the Credential and Batch Credential Response on top of TLS. If this object is present, the Credential Issuer expects Credential and Batch Credential Response to be encrypted. If this object is omitted, Credential  and Batch Credential Response should not be encrypted.
```

Comment:
```suggestion
* `credential_response_encryption`: OPTIONAL. Object containing information whether the Credential Issuer requires encryption of the Credential and Batch Credential Response on top of TLS. If this object is present, the Credential Issuer expects Credential and Batch Credential Response to be encrypted. If this object is omitted, Credential  and Batch Credential Response should not be encrypted. If present, the object consists of exactly two parameters:
```

Reply:
not sure why this is needed..?

Reply:
I think you requested this in another conversation in this PR actually.